Buildings/Fluid/HeatExchangers/RadiantSlabs/SingleCircuitSlab.mo

```diff
@@ -226,6 +226,10 @@ user's guide</a> for more information.
 revisions="<html>
 <ul>
 <li>
+January 06, 2016, by Thierry S. Nouidui:<br/>
+Renamed parameter <code>nSta2</code> to <code>nSta</code>.
```

Comment:
I might have merged the master in at some point by mistake before I've merged ```issue609_cdl```. I'll revert these back to the 609 state.